src/Exception/ConfigurationException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FriendsOfPhpSpec\PhpSpec\CodeCoverage\Exception;
+
+use InvalidArgumentException;
+
+class ConfigurationException extends InvalidArgumentException
+{
+}
```

Comment:
Does this have any benefit over built-in exceptions, like `\RuntimeException('Invalid configuration')`?

Reply:
It depends I've refrain to even add an exception marker in a previous commit I went with PHP built in exception. I can remove the new Exception class if needed. It was a way to express the nature of the exception. Either way I'm fine with your call. To keep it or to remove it

Reply:
Should I remove it or is it fine for you to keep it ?

Reply:
I'll let @shulard and @drupol decide.

Reply:
Hello !

I also like those expressive exceptions ^^. What do you think @drupol  ?

Reply:
Ok I saw that everyone agreed on those changes. Let merge them now ! Thank you @nyamsprod for the update !